examples/basics/emag_basics.py

```diff
@@ -3,220 +3,249 @@
+mc.set_variable("Shaft_Speed_[RPM]", 1000)
+mc.set_variable("CurrentDefinition", 0)
+mc.set_variable("PeakCurrent", 3)
+mc.set_variable("DCBusVoltage", 350)
+mc.set_variable("PhaseAdvance", 45)
 
 # %%
-# Save the file.
-filename = os.path.join(working_folder, "../ActiveX_Scripting_EMagnetic.mot")
-mcad.save_to_file(filename)
+# Save the file
+# ~~~~~~~~~~~~~
+# Save the file to the temporary folder and display a message that the initialisation in complete.
+mc.save_to_file(working_folder + "/" + mot_name + ".mot")
+print("Initialisation completed.")
 
 # %%
 # Run simulation
 # --------------
-# Run the simulation.
-mcad.do_magnetic_calculation()
+# Run the electromagnetic calculation in Motor-CAD (solve the e-magnetic model).
+print("Running simulation.")
+mc.do_magnetic_calculation()
 
 # %%
 # Export results to CSV file
 # --------------------------
-# Export results to a CSV file.
-exportFile = os.path.join(working_folder, "../Export_EMag_Results.csv")
+# Export results to a CSV file. Results will be saved as *Export_EMag_Results.csv* in the temporary
+# working directory. A try and except block are used to raise an error in the case of the export
+# failing.
 try:
-    mcad.export_results("EMagnetic", exportFile)
+    mc.export_results("EMagnetic", working_folder + "/Export_EMag_Results.csv")
     print("Results successfully exported.")
-except pymotorcad.MotorCADError:
-    print("Results failed to export.")
+except pymotorcad.MotorCADError as e:
+    print("Results failed to export due to Motor-CAD Error: " + str(e))
 
 # %%
-# Get and analyze results
+# Get and analyse results
 # -----------------------
-# Get torque and voltage data.
-shaft_torque = mcad.get_variable("ShaftTorque")
-line_voltage = mcad.get_variable("PeakLineLineVoltage")
+# Retrieve the torque and voltage output data values.
+shaft_torque = mc.get_variable("ShaftTorque")
+line_voltage = mc.get_variable("PeakLineLineVoltage")
+print(
+    f"Shaft Torque: {shaft_torque:.2f} Nm, Line-Line Terminal Voltage (peak): {line_voltage:.2f} V"
+)
 
 # %%
-# Graph the torque data.
-num_torque_points = points_per_cycle * number_cycles
-rotor_position = []
-torque_vw = []
-
-for n in range(num_torque_points):
-    (x, y) = mcad.get_magnetic_graph_point("TorqueVW", n)
-    rotor_position.append(x)
-    torque_vw.append(y)
+# Retrieve the *Torque (VW)* graph data. The data type for this series is *E-Magnetics*, so use the
+# ``get_magnetic_graph()`` method.
+#
+# .. note::
+#    When retrieving graph data points, use the *Graph Viewer* under *Help* in the Motor-CAD
+#    interface to find the series names and data types.
+rotor_position, torque_vw = mc.get_magnetic_graph("TorqueVW")
 
 # %%
-# Graph the airgap flux density data.
-loop = 0
-success = 0
-mech_angle = []
-airgap_flux_density = []
+# Retrieve the airgap flux density graph data. This data can be found in the *Graph Viewer* in
+# Motor-CAD. The data type for this series is *FEA Path*, so use the ``get_fea_graph()`` method.
```

Comment:
```suggestion
# Retrieve the airgap flux density graph data. This data can be found in the **Graph Viewer** in
# Motor-CAD. The data type for this series is **FEA Path**, so use the ``get_fea_graph()`` method.
```
Not sure if **FEA Path** in bold for UI element or double backticks for code entity?